test/bigtest/tests/sort-collection-test.js

```diff
@@ -174,8 +157,6 @@ it('sorts correctly', () => {
     '-number',
     sortByDates,
   ];
-  const sortFunctionResult = sortCollection(collectionUnsorted, sortingOptions);
-  const isSortedCorrectly = isEqual(sortFunctionResult, collectionSorted);
 
-  expect(isSortedCorrectly).to.be.true;
+  expect(sortCollection(collectionUnsorted, sortingOptions)).to.deep.equal(collectionSorted);
```

Comment:
If it is possible to avoid deep comparison let's avoid it. I see that previously `is.equal` worked just fine. Has something changed?

Reply:
I don't think it is possible to do it without deep comparison. Previously it was compared with `lodash.isEqual` function and I decided that is is better to use `.deep.equal` from chai

Reply:
Sounds good.